### ISSUE TITLE
Remove aria-hidden attribute from icons 

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/Icons#137]
## Is your feature request related to a problem? Please describe.

Due to an opinion that all icons are to be hidden from screen readers, it has come to our attention that this is not an opinion we should be enforcing. 

## Describe the solution you'd like

Remove `aria-hidden="true"` attribute from all icons. This can be done by removing the following line of code:

https://github.com/AlaskaAirlines/Icons/blob/603c36f0074daa529b57e2dee2810f3a56e30c43/src/data/icons.json#L4

## Additional context

- https://github.com/AlaskaAirlines/auro-icon/issues/46
